RubberduckTests/Binding/SimpleNameDefaultBindingTests.cs

```diff
@@ -18,20 +18,21 @@ public class SimpleNameDefaultBindingTests
         [TestMethod]
         public void EnclosingProcedureComesBeforeEnclosingModule()
         {
-            string testCode = string.Format(@"
```

Comment:
It was correctly not resolving the types because there weren't any references to it once I fixed the resolver error from the invalid code being tested against.  I made it so there was one reference again.